scipy/spatial/tests/test_kdtree.py

```diff
@@ -873,7 +873,7 @@ def test_kdtree_build_modes(kdtree_type):
 def test_kdtree_pickle(kdtree_type):
     # test if it is possible to pickle a KDTree
     try:
-        import cPickle as pickle
+        import cPickle as pickle  # type: ignore[import]
```

Comment:
I think `cPickle` importing is irrelevant for the newer version of Python `3.x` that we support--the performance advantages have been merged into plain `pickle`--so, we could clean that up at some point as well.